sci-libs/tensorflow/tensorflow-2.13.1.ebuild

```diff
@@ -0,0 +1,453 @@
+	)
+	python? (
+		https://github.com/intel/ARM_NEON_2_x86_SSE/archive/a15b489e1222b2087007546b4912e21293ea86ff.tar.gz -> ARM_NEON_2_x86_SSE-a15b489e1222b2087007546b4912e21293ea86ff.tar.gz
+		https://storage.googleapis.com/mirror.tensorflow.org/docs.python.org/2.7/_sources/license.rst.txt -> tensorflow-1.15.0-python-license.rst.txt
+	)"
+
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz
+		${bazel_external_uris}"
+
+# absl/log/check.h is needed in tensorflow-2.13.1, see
+# https://github.com/tensorflow/tensorflow/blob/v2.13.1/tensorflow/compiler/jit/xla_compile_on_demand_op.cc
+# which requires abseil-cpp>=20230125, see
+# https://github.com/abseil/abseil-cpp/commit/92fdbfb301f8b301b28ab5c99e7361e775c2fb8a
+
+# abseil-cpp need to compile with C++17
+# abseil-cpp>=20230125.3 in repo are built with C++14
+
+# check flatbuffers version in tensorflow/lite/schema/schema_generated.h
+
+# BDEPEND: >=dev-libs/protobuf-3.8.0
+		#>=dev-cpp/abseil-cpp-20230125.0:=
+RDEPEND="
+	app-arch/snappy
+	=dev-cpp/abseil-cpp-20230125.2*:=
+	dev-db/sqlite
+	dev-libs/double-conversion
+	dev-libs/icu:=
+	>=dev-libs/jsoncpp-1.9.2:=
+	>=dev-libs/nsync-1.25.0
+	dev-libs/openssl:0=
+	>=dev-libs/protobuf-3.13.0:=
+	>=dev-libs/re2-0.2019.06.01:=
+	media-libs/giflib
+	media-libs/libjpeg-turbo
+	media-libs/libpng:0
+	>=net-libs/grpc-1.28:=
+	net-misc/curl
+	sys-libs/zlib
+	>=sys-apps/hwloc-2:=
+	cuda? (
+		dev-util/nvidia-cuda-toolkit:=[profiler]
+		=dev-libs/cudnn-8*
+	)
+	mpi? ( virtual/mpi )
+	python? (
+		${PYTHON_DEPS}
+		~dev-libs/flatbuffers-23.1.21:=
+		dev-python/absl-py[${PYTHON_USEDEP}]
+		>=dev-python/astor-0.7.1[${PYTHON_USEDEP}]
+		dev-python/astunparse[${PYTHON_USEDEP}]
+		dev-python/clang-python[${PYTHON_USEDEP}]
+		dev-python/dill[${PYTHON_USEDEP}]
+		~dev-python/flatbuffers-23.1.21[${PYTHON_USEDEP}]
+		>=dev-python/gast-0.3.3[${PYTHON_USEDEP}]
+		dev-python/h5py[${PYTHON_USEDEP}]
+		>=dev-python/numpy-1.19[${PYTHON_USEDEP}]
+		>=dev-python/google-pasta-0.1.8[${PYTHON_USEDEP}]
+		>=dev-python/opt-einsum-3.3.0[${PYTHON_USEDEP}]
+		>=dev-python/protobuf-python-3.13.0[${PYTHON_USEDEP}]
+		dev-python/pybind11[${PYTHON_USEDEP}]
+		dev-python/six[${PYTHON_USEDEP}]
+		dev-python/tblib[${PYTHON_USEDEP}]
+		dev-python/termcolor[${PYTHON_USEDEP}]
+		dev-python/typing-extensions[${PYTHON_USEDEP}]
+		>=dev-python/grpcio-1.28[${PYTHON_USEDEP}]
+		>=dev-python/wrapt-1.11.1[${PYTHON_USEDEP}]
+		>=net-libs/google-cloud-cpp-0.10.0
+		=sci-visualization/tensorboard-${DEP_VER}*[${PYTHON_USEDEP}]
+	)"
+DEPEND="${RDEPEND}
+	python? (
+		dev-python/mock
+		dev-python/setuptools
+	)"
+PDEPEND="python? (
+		=sci-libs/keras-${DEP_VER}*[${PYTHON_USEDEP}]
+		=sci-libs/tensorflow-estimator-${DEP_VER}*[${PYTHON_USEDEP}]
+	)"
+BDEPEND="
+	app-arch/unzip
+	=dev-build/bazel-5*
+	dev-java/java-config
+	cuda? (
+		>=dev-util/nvidia-cuda-toolkit-9.1[profiler]
+	)
+	!python? ( dev-lang/python )
```

Comment:
This looks weird. It is almost always wrong to depend on python directly instead of `${PYTHON_DEPS}`